### ISSUE TITLE
Document component capable of rendering markdown/code

### ISSUE THREAD

**Issue by nimishbongale** [cc-archive/vocabulary-legacy#735]
## Description
A markdown rendering component would be really handy to have. We would then be able to render any other code snippets as well. Writing docs would also become a much smoother experience. 

## Alternatives

- https://github.com/miaolz123/vue-markdown
- https://developer.aliyun.com/mirror/npm/package/vue-markdown-loader

The community led versions may not have the option to customize the font-family according to our requirements. This remains to be seen and discussed. 

## Implementation

- [x] I would be interested in implementing this feature.

**Comment by JipSterk** [comment]
i would like to try and implement this

**Comment by nimishbongale** [comment]
Go ahead @JipSterk!

**Comment by zackkrida** [comment]
@JipSterk unrelated, but i love your github avatar

**Comment by nimishbongale** [comment]
@zackkrida @dhruvkb 

Just to reiterate over this issue:

I did consider having this would be a great addition for us, but I now understand that the use cases would be rather limited. I've used [vue-markdown](https://github.com/miaolz123/vue-markdown), and I also overrode it with our own custom styles. It seems to work great!

Here's a screenshot:

![Screenshot 2020-11-21 112124](https://user-images.githubusercontent.com/43414361/99868795-bfcb1b00-2beb-11eb-90da-fd5d792d3549.png)

Maybe, instead of having it as a separate component (which would be a near enough replica of vue-markdown, and would bloat our library size) we could just document it under the `code` typography section?
